Entity/ScheduledTask.php

```diff
@@ -38,7 +37,7 @@ class ScheduledTask implements StatusInterface
     /**
      * @var string
      *
-     * @ORM\Column(name="name", type="string", length=255, options={"comment":"Scheduled task command name"})
+     * @ORM\Column(name="name", type="string", length=2048, options={"comment":"Scheduled task command name"})
```

Comment:
Why do you need 2048 chars length?

Reply:
Actually, we moved to full text and already did a patch from the internally forked version. Because we had this issue in summer.